### ISSUE TITLE
IndexShardWriterImpl.java has a hardcoded limit of 1s before WARNing of a slow addDocument() call

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#2855]
Can this please have a configurable limit

**Comment by at055612** [comment]
Now logs
```
WARN  2022-04-29T12:10:21.379Z Data Processor# #1 s.i.i.IndexShardWriterImpl     addDocument() - (id=1) took PT0.065060934S (Warning threshold: PT0.0001s. Configure this with property stroom.index.writer.slowIndexWriteWarningThreshold) 
WARN  2022-04-29T12:10:21.381Z Data Processor# #1 s.i.i.IndexShardWriterImpl     addDocument() - (id=1) took PT0.000458721S (Warning threshold: PT0.0001s. Configure this with property stroom.index.writer.slowIndexWriteWarningThreshold) 
WARN  2022-04-29T12:10:21.381Z Data Processor# #1 s.i.i.IndexShardWriterImpl     addDocument() - (id=1) took PT0.000375481S (Warning threshold: PT0.0001s. Configure this with property stroom.index.writer.slowIndexWriteWarningThreshold) 
WARN  2022-04-29T12:10:21.382Z Data Processor# #1 s.i.i.IndexShardWriterImpl     addDocument() - (id=1) took PT0.00035492S (Warning threshold: PT0.0001s. Configure this with property stroom.index.writer.slowIndexWriteWarningThreshold) 
```

Default remains at `PT1S`

**Comment by at055612** [comment]
Fixed in >7.0-beta.200
